### PR TITLE
Add Lemon Squeezy icon

### PR DESCRIPTION
**Pull request by CodeShakingSheep**
image

**Issue:** closes https://github.com/simple-icons/simple-icons/issues/10890

**Popularity metric:**

The Similarweb rank is 31,610. See https://www.similarweb.com/website/lemonsqueezy.com

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

I downloaded the official logo svg from [here](https://assets-global.website-files.com/6347244ba8d63489ba51c08e/6347244ba8d6344eb751c0d5_Logo.svg). Then I removed the letters, resized the canvas, compounded the path, rearranged it, optimized it.

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -8503,6 +8503,11 @@
                 "url": "https://github.com/LemmyNet/lemmy/blob/main/LICENSE"
             }
         },
+        {
+            "title": "Lemonsqueezy",
```
Comment (uncenter) [inline _data/simple-icons.json:8507]:
```suggestion
            "title": "Lemon Squeezy",
```

File: icons/lemonsqueezy.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Lemonsqueezy</title><path d="m7.4916 10.835 2.3748-6.5114a3.1497 3.1497 0 0 0-.065-2.3418C9.0315.183 6.9427-.398 5.2928.265 3.643.929 2.71 2.4348 3.512 4.3046l2.8197 6.5615c.219.509.97.489 1.16-.03m1.6798 1.0969 6.5334-2.7758c2.1699-.9219 2.7218-3.6907 1.022-5.2905l-.068-.063c-1.6669-1.5469-4.4217-1.002-5.3706 1.0359L8.3566 11.135c-.234.503.295 1.0199.8159.7979m.373.87 6.6454-2.5119c2.2078-.8349 4.6206.745 4.5886 3.0398l-.002.09c-.048 2.2358-2.3938 3.7376-4.5536 2.9467l-6.6724-2.4418a.595.595 0 0 1-.006-1.1229m-.386 1.9269 6.4375 2.9767a3.2997 3.2997 0 0 1 1.6658 1.6989c.769 1.7998-.283 3.6396-1.9328 4.3016-1.6499.662-3.4097.235-4.2097-1.6359l-2.8027-6.5694c-.217-.509.328-1.009.8419-.772"/></svg>
```
Comment (uncenter) [inline icons/lemonsqueezy.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Lemon Squeezy</title><path d="m7.4916 10.835 2.3748-6.5114a3.1497 3.1497 0 0 0-.065-2.3418C9.0315.183 6.9427-.398 5.2928.265 3.643.929 2.71 2.4348 3.512 4.3046l2.8197 6.5615c.219.509.97.489 1.16-.03m1.6798 1.0969 6.5334-2.7758c2.1699-.9219 2.7218-3.6907 1.022-5.2905l-.068-.063c-1.6669-1.5469-4.4217-1.002-5.3706 1.0359L8.3566 11.135c-.234.503.295 1.0199.8159.7979m.373.87 6.6454-2.5119c2.2078-.8349 4.6206.745 4.5886 3.0398l-.002.09c-.048 2.2358-2.3938 3.7376-4.5536 2.9467l-6.6724-2.4418a.595.595 0 0 1-.006-1.1229m-.386 1.9269 6.4375 2.9767a3.2997 3.2997 0 0 1 1.6658 1.6989c.769 1.7998-.283 3.6396-1.9328 4.3016-1.6499.662-3.4097.235-4.2097-1.6359l-2.8027-6.5694c-.217-.509.328-1.009.8419-.772"/></svg>
```

**Review by uncenter — CHANGES_REQUESTED** [review]

File: icons/lemonsqueezy.svg (hunk 2 above)
Comment (uncenter) [reply icons/lemonsqueezy.svg:1]: (Other than this though the icon itself looks great/accurate!)

Comment (CodeShakingSheep) [reply icons/lemonsqueezy.svg:1]: Hi @uncenter, Thanks for reviewing. You're right. I mixed up the spelling.

**Review by uncenter — APPROVED** [review]

**Pull request merged**